graylog2-web-interface/src/views/components/contexts/FieldTypesProvider.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow strict
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import { useStore } from 'stores/connect';
+import { FieldTypesStore } from 'views/stores/FieldTypesStore';
+import FieldTypesContext from './FieldTypesContext';
+
+const FieldTypesProvider = ({ children }: { children: React.Node }) => {
+  const fieldTypes = useStore(FieldTypesStore);
+  return (
+    fieldTypes
+      ? (
+        <FieldTypesContext.Provider value={fieldTypes}>
+          {children}
+        </FieldTypesContext.Provider>
+      )
+      : children
+  );
+};
```

Comment:
We had this discussion for e.g. the `CurrentUserProvider`: Right now I am expecting `fieldTypes` to be `undefined` initially, which means we need to test / check the `undefined` case in every component where we are using the `FieldTypesContext`.